bnb_yaz/bnb-downloader.py

```diff
@@ -2,262 +2,425 @@
 # -*- coding: utf-8 -*-
 #
 # A tool for the automation of MARC downloads from the BNB.
+# This version reads BNB IDs from a PDF file and downloads records from the BNB server.
+# Author: Steven Lolong (steven.lolong@uni-tuebingen.de)
+# First created: November 2025
+# Usage: new_bnb_downloader.py <email> <working_directory>
```

Comment:
This usage documentation here is duplicate / outdated and should be removed from the head comment